### ISSUE TITLE
Decide on a cool name for the Django service

### ISSUE THREAD

**Issue by MgenGlder** [dearborn-coding-club/website-base-backend#11]
## Summary

It feels kind of weird to keep calling the backend service the "Django App". I think it would be better to come up with some kind of witty name that represents what we want to do with it. Recommendations welcome.

**Comment by gonzalolanderos** [comment]
Maybe we should start by defining what we want to do with it. Either that or have something completely abstract to have a label. As for the functionality, is the idea to setup different types of projects within Django that will help us understand Python better?

**Comment by MgenGlder** [comment]
> Maybe we should start by defining what we want to do with it.

Fair point- I'm thinking we use it as an orchestrator service. It's purpose should be to route requests to the main website, as well as delegate to other services for functions/operations it doesn't know how to do. Kind of like a core monolithic backend that reaches out to single purpose microservices when relevant.

> As for the functionality, is the idea to setup different types of projects within Django that will help us understand Python better?

We can certainly go that route- my original thought was to use the Django app to build out some interesting functionality for the site (interacting with `meetup.com` api's, etc). For more specialized work like integrating with a Dart/Flutter phone app, we could set up other microservices in other languages that live on the same cluster. 

Ideally this project would be a choose-your-own-adventure type of deal, where you could work on Python/Django if you want, but there's opportunity to work on Rust, Go, Java/Kotlin, Dart/Flutter etc based on interest.
